packages/docusaurus/bin/docusaurus.js

```diff
@@ -55,10 +55,12 @@ program
   });
 
 program
-  .command('eject [siteDir]')
-  .description('copy the default theme into website folder for customization.')
-  .action((siteDir = '.') => {
-    wrapCommand(eject)(path.resolve(siteDir));
+  .command('eject <pluginName> [siteDir] ')
+  .description(
+    'copy the plugin theme files into website folder for customization.',
```

Comment:
Title case please - "Copy the theme ...". And let's call them themes and not plugins.